module_utils/storage_lsr/argument_validator.py

```diff
@@ -7,6 +7,26 @@
 # pylint: disable=undefined-variable
 STRING_TYPE = str if sys.version_info.major == 3 else basestring  # noqa:F821
 
+BOOLEANS_TRUE = ['y', 'yes', 'on', '1', 'true', 't', 1, 1.0, True]
```

Comment:
I think this should be a separate commit with some comment, this doesn't look related to the thin provisioning support.

Reply:
Fixed.